microsoft-edge/web-platform/how-to-detect-win11.md

```diff
@@ -82,6 +82,41 @@ navigator.userAgentData.getHighEntropyValues(["platformVersion"])
 
 ```
 
+<!-- ====================================================================== -->
+## Sample code for detecting ARM or x86
+
+Detection of CPU architecture is helpful for websites that wish to guide visitors to download applications built for the CPU in their system. This is particularly helpful for ARM-based devices so that customers download the native ARM version of an application, instead of inadvertently installing an app built for x86 and experiencing reduced performance from emulation.
```

Comment:
```suggestion
Use detection of CPU architecture to have your website automatically download the version of your app that's built specifically for the user's CPU.

CPU detection is particularly helpful for ARM-based devices, so that a customer using an ARM device automatically downloads the native ARM version of an application.  This prevents the user from inadvertently installing an app that's built for x86, and then experiencing reduced performance due to emulation.
```
Writer pass:
* broke up long sentence per Acrolinx.
* singular instead of plural, to reduce ambiguity.